extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchMergeAggregatorFactory.java

```diff
@@ -86,7 +86,7 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
           name,
           Math.max(size, castedOther.size),
           shouldFinalize,
-          true,
+          false,
```

Comment:
isInputThetaSketch is used only at indexing time when reading user's input data to decide if its [base64 encoded] sketch objects. it has no role to play afterwards.
that value in merged factory does not change anything as long as it stays same, just thought false would be better to store there. it could stay true as well without any problems.